src/pages/DetailsPage.js

```diff
@@ -37,12 +40,14 @@ const DetailsPage = ({personalDetails, route}) => {
         <ScreenWrapper>
             <HeaderWithCloseButton
                 title="Details"
+                shouldShowBackButton={!!route.params.reportID}
```

Comment:
This feels like an implementation detail. Can we make this more explicit by adding something above like:

```js
// If we have a reportID param this means that we arrived here via the ParticipantsPage and should be allowed to navigate back to it
const shouldShowBackButton = Boolean(route.params.reportID);
```